code/obj/machinery/alarm.dm

```diff
@@ -3,24 +3,37 @@
 //
 
 /obj/machinery/alarm
-	name = "Air Monitor"
+	name = "air Monitor"
 	icon = 'icons/obj/monitors.dmi'
-	icon_state = "alarm0"
+	icon_state = "alarm_unpowered"
 	power_usage = 5
 	power_channel = ENVIRON
 	anchored = ANCHORED
-	var/skipprocess = 0 //Experimenting
+	/// save some CPU by only checking every tick when something is amiss
+	var/skipprocess = 0
 	var/alarm_frequency = FREQ_ALARM
 	var/alarm_zone = null
 	var/control_frequency = FREQ_AIR_ALARM_CONTROL
-	var/id
-	var/locked = 1
-
-	var/datum/gas_mixture/environment
-	var/safe
-	/*var/panic_mode = 0 */
-	var/e_gas = 0
+	/// keeps track of last alarm status
 	var/last_safe = 2
+	var/datum/gas_mixture/environment
+
+	//this is a list of safe & good partial pressures of each gas. If all gasses are in the good range, the alarm will show green. If any gas is outside the safe range, the alarm will show alert. Otherwise caution.
+	//most of these values are taken from lung.dm
```

Comment:
```suggestion
	/// this is a list of safe & good partial pressures of each gas. If all gasses are in the good range, the alarm will show green. If any gas is outside the safe range, the alarm will show alert. Otherwise caution.
	/// most of these values are taken from lung.dm
```